client/app/components/preprint-basics/component.js

```diff
@@ -66,30 +68,22 @@ export default Ember.Component.extend(BASICS_VALIDATIONS, {
     }),
     basicsDOI: null,
     basicsLicense: null,
+    savedValues: null,
 
-    /* Check if discard is needed */
-    savedValues: Ember.computed('node', function() {
-        let node = this.get('node');
-        let values = {
-            basicsDOI: null,
-            basicsLicense: null,
-            basicsTags: node.get('tags').map(fixSpecialChar),
-            basicsAbstract: node.get('description')
-        }
-        return values ;
-    }),
-    basicsChanged:  Ember.computed('basicsDOI', 'basicsLicense', 'basicsTags.@each', 'basicsAbstract', 'applyLicense', function() {
+    basicsChanged:  Ember.observer('savedValues', 'basicsDOI', 'basicsLicense', 'basicsTags.@each', 'basicsAbstract', 'applyLicense', function() {
+        let changed = false;
         let saved = this.get('savedValues');
-        let doiChanged = saved.basicsDOI !== this.get('basicsDOI');
-        let licenseChanged = saved.basicsLicense !== this.get('basicsLicense') && this.get('applyLicense');
-        let abstractChanged = saved.basicsAbstract ? saved.basicsAbstract !== this.get('basicsAbstract') : false ;
-        let tagsChanged = saved.basicsTags ? saved.basicsTags.length !== this.get('basicsTags').length || saved.basicsTags.some((v,i) => v !== this.get('basicsTags')[i])  : false ;
-
-      return doiChanged || licenseChanged || abstractChanged || tagsChanged;
+        if (saved !== null) {
+            let doiChanged = saved.basicsDOI !== this.get('basicsDOI');
+            let licenseChanged = saved.basicsLicense !== this.get('basicsLicense') && this.get('applyLicense');
+            let abstractChanged = saved.basicsAbstract ? saved.basicsAbstract !== this.get('basicsAbstract') : false;
+            let tagsChanged = saved.basicsTags ? saved.basicsTags.length !== this.get('basicsTags').length || saved.basicsTags.some((v, i) => v !== this.get('basicsTags')[i]) : false;
+            changed = doiChanged || licenseChanged || abstractChanged || tagsChanged;
```

Comment:
I tried making this a computed property, and it does not update after the section closes because the view-only/closed template does not contain `basicsChanged`. I'm not sure where calling this.get would work around that issue.